.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Python CI
+
+on:
+   push:
+     branches: [ master ]
+   pull_request:
+     branches:
+       - '**'
+
+
+jobs:
+   run_tests:
+     name: tests
+     runs-on: ${{ matrix.os }}
+     strategy:
+       matrix:
+         os: [ubuntu-20.04]
+         python-version: ['3.5','3.8']
+         toxenv: ['django22', 'quality']
+         exclude:
+           - python-version: '3.5'
+             toxenv: 'quality'
+
+     steps:
+       - uses: actions/checkout@v2
```

Comment:
you might want to switch to using `actions/checkout@v1`, we've faced issues with coverage checks in some other repos with checkout v2

Reply:
Should I make this change in all repos I have added GitHub Actions to? Or its specific to this one?

Reply:
in all the repos where we've coverage checks at least

Reply:
Okay got it